crates/rspack_build/fixture/test.config.json

```diff
@@ -1,7 +1,12 @@
 {
   "entry": {
     "index": {
-      "import": ["./app.js"]
+      "import": [
+        "./app.js"
+      ]
     }
+  },
+  "output": {
+    "clear": true
```

Comment:
tests should be added in `packages/rspack/tests`

Reply:
I put the files in the packages/rspack/tests/Output.test.ts